projects/fishtornado_xr/FishTornado.cpp

```diff
@@ -1057,9 +1074,10 @@ void FishTornadoApp::Render()
         submitInfo.pFence = frame.uiRenderCompleteFence;
 
         PPX_CHECKED_CALL(GetGraphicsQueue()->Submit(&submitInfo));
+        PPX_CHECKED_CALL(uiSwapchain->Present(uiImageIndex, 0, nullptr));
     }
 
-    grfx::SwapchainPtr swapchain = GetSwapchain(currentViewIndex);
+    grfx::SwapchainPtr swapchain = GetSwapchain(mViewIndex);
 
     UpdateTime();
 
```

Comment:
sorry i am not able to add comment to the line directly, but the comment in line 1069 is not valid anymore with this change, so the comment needs to be be changed.